gtk4/src/subclass/accessible_text.rs

```diff
@@ -0,0 +1,533 @@
+
+            let func = (*parent_iface)
+                .get_offset
+                .expect("no parent \"get_offset\" implementation");
+
+            let mut offset = MaybeUninit::uninit();
+
+            let offset_set = from_glib(func(
+                self.obj()
+                    .unsafe_cast_ref::<AccessibleText>()
+                    .to_glib_none()
+                    .0,
+                point.to_glib_none().0,
+                offset.as_mut_ptr(),
+            ));
+
+            if offset_set {
+                Some(offset.assume_init())
+            } else {
+                None
+            }
+        }
+    }
+
+    fn parent_selection(&self) -> Vec<AccessibleTextRange> {
+        unsafe {
+            let type_data = Self::type_data();
+            let parent_iface = type_data.as_ref().parent_interface::<AccessibleText>()
+                as *const ffi::GtkAccessibleTextInterface;
+
+            let func = (*parent_iface)
+                .get_selection
+                .expect("no parent \"get_selection\" implementation");
+
+            let mut n_ranges = MaybeUninit::uninit();
+            let mut ranges = std::ptr::null_mut();
+
+            let valid = from_glib(func(
+                self.obj()
+                    .unsafe_cast_ref::<AccessibleText>()
+                    .to_glib_none()
+                    .0,
+                n_ranges.as_mut_ptr(),
+                &mut ranges,
+            ));
+
+            if valid {
+                let n = n_ranges.assume_init();
+                AccessibleTextRange::from_glib_container_num_as_vec(ranges, n)
+            } else {
+                Vec::new()
+            }
+        }
+    }
+}
+
+impl<T: AccessibleTextImpl> AccessibleTextImplExt for T {}
+
+unsafe impl<T: AccessibleTextImpl> IsImplementable<T> for AccessibleText {
+    fn interface_init(iface: &mut glib::Interface<Self>) {
+        let iface = iface.as_mut();
+
+        iface.get_contents = Some(accessible_text_get_contents::<T>);
+        iface.get_contents_at = Some(accessible_text_get_contents_at::<T>);
+        iface.get_caret_position = Some(accessible_text_get_caret_position::<T>);
+        iface.get_selection = Some(accessible_text_get_selection::<T>);
+        iface.get_attributes = Some(accessible_text_get_attributes::<T>);
+        iface.get_default_attributes = Some(accessible_text_get_default_attributes::<T>);
+
+        #[cfg(feature = "v4_16")]
+        {
+            iface.get_extents = Some(accessible_text_get_extents::<T>);
+            iface.get_offset = Some(accessible_text_get_offset::<T>);
+        }
+    }
+}
+
+unsafe extern "C" fn accessible_text_get_contents<T: AccessibleTextImpl>(
+    accessible_text: *mut ffi::GtkAccessibleText,
+    start: libc::c_uint,
+    end: libc::c_uint,
+) -> *mut glib::ffi::GBytes {
+    let instance = &*(accessible_text as *mut T::Instance);
+    let imp = instance.imp();
+
+    let contents = imp.contents(start, end);
+    contents.to_glib_full()
```

Comment:
```suggestion
    contents.into_glib_ptr()
```

also for all other `to_glib_full()` here